File: Java/Car.java
import com.sun.tools.classfile.Annotation.element_value;

class Car
{
    Integer id;
    String license;
    Account driver;
    private Integer passegenger;

    public Car(String pLicense, Account pDriver){
        this.license = pLicense;
        this.driver = pDriver;
    }

    void printDataCar(){
        if(passegenger!=null)
            System.out.println("License:" + license + " Name Driver: " + driver.name + " Passengers: " + passegenger);

    }

    public Integer getPassenger(){
        return passegenger;
    }

    public void setPassenger(Integer pPassenger){
        if(pPassenger==4)
        this.passegenger = pPassenger;
        else
        System.out.println("Pasasegos no válidos.");
    }

}

File: PHP/Car.php
<?php
require_once 'Account.php';
class Car{
    public $id;
    public $license;
    public $driver;
    public $passegenger;

    public function __construct($pLicense, $pDriver){
        $this->license = $pLicense;
        $this->driver=$pDriver;
    }

    public function printDataCarAsString(){
        //return "License: " . $license . " Name driver: " . $driver;
        echo "Licencia: $this->license Driver: " . $this->driver->name;
    }

}
